docs/azure/sdk/authentication/additional-methods.md

```diff
@@ -36,6 +51,122 @@ foreach (var blobItem in client.GetBlobContainers())
+
+```csharp
+using Azure.Identity;
+using Azure.Identity.Broker;
+
+private void button1_Click(object sender, EventArgs e)
+{
+    // Get the handle of the current window
+    IntPtr windowHandle = this.Handle;
+
+    var credential = new InteractiveBrowserCredential(
+        new InteractiveBrowserCredentialBrokerOptions(windowHandle));
+
+    // To authenticate and authorize with an app, substitute
+    // <tenant-id> and <client-id> with the values for your app and tenant.
+    // var credential = new InteractiveBrowserCredential(
+    //    new InteractiveBrowserCredentialBrokerOptions(windowHandle)
+    //        { 
+    //            TenantId = "<tenant-id>",
+    //            ClientId = "<client-id>"
+    //        }
+    // );
+
+    var client = new BlobServiceClient(
+        new Uri("https://<storage-account-name>.blob.core.windows.net/"),
+        credential
+    );
+
+    // Prompt for credentials appears on first use of the client
+    foreach (var container in client.GetBlobContainers())
+    {
+        Console.WriteLine(container.Name);
+    }
+}
+```
+
+> [!NOTE]
+> Visit the [Parent window handles](/entra/msal/dotnet/acquiring-tokens/desktop-mobile/wam#parent-window-handles) and [Retrieve a window handle](/windows/apps/develop/ui-input/retrieve-hwnd) articles for more information about retrieving window handles.
+
+For the code to run successfully, your user account must be assigned an Azure role on the storage account that allows access to blob containers such as **Storage Account Data Contributor**. If an app is specified, it must have API permissions set for **user_impersonation Access Azure Storage** (step 6 in the previous section). This API permission allows the app to access Azure storage on behalf of the signed-in user after consent is granted during sign-in.
+
+The following screenshot shows the user sign-in experience:
+
+:::image type="content" source="../media/web-account-manager-sign-in-account-picker.png" alt-text="A screenshot that shows the sign-in experience when using the interactive browser broker credential to authenticate a user." :::
+
+### Authenticate the default system account via WAM
+
+Many people always sign in to Windows with the same user account and, therefore, only ever want to authenticate using that account. WAM and `InteractiveBrowserCredential` also support a silent login process that automatically uses a default account so the user does not have to repeatedly select it.
+
+The following example shows how to enable sign-in with the default system account:
+
+```csharp
+using Azure.Identity;
+using Azure.Identity.Broker;
+
+// App code omitted for brevity
+
+IntPtr windowHandle = this.Handle;
+
+var credential = new InteractiveBrowserCredential(
+    new InteractiveBrowserCredentialBrokerOptions(windowHandle)
+    {
+        // Enable silent brokered authentication using the default account
+        UseDefaultBrokerAccount = true,
```

Comment:
I recommend creating a sample app to accompany this doc. Then you can import code snippets and highlight important lines of code like this one.